src/transformers/generation_utils.py

```diff
@@ -612,6 +613,19 @@ def _get_logits_processor(
             processors.append(ForcedEOSTokenLogitsProcessor(max_length, forced_eos_token_id))
         return processors
 
+    def _get_stopping_criteria(
+        self,
+        max_length: Optional[int],
+        max_time: Optional[float],
+    ) -> StoppingCriteriaList:
+
```

Comment:
Can we add some:

```
max_length = max_length if max_length is not None else max_length
```

same for `max_time` 

and remove where it's done earlier?

Reply:
I can't for `max_length` (because of the other `max_length` usages) and deprecation. 
I'll do if you still want `max_time` to be in the config.